scripts/tests

```diff
@@ -360,6 +366,28 @@ def bench(force_qemu, verbose, cycles, cflags, arch_flags):
         [f"CYCLES={cycles}"],
     )
 
+    if output:
+        import json
+
+        with open(output, "w") as f:
+            v = []
+            for scheme in results:
+                schemeStr = str(scheme)
+                r = results[scheme]
+                d = {
+                    k: int(v)
+                    for k, v in (l.decode().split("=") for l in r.splitlines()[:3])
```

Comment:
A short comment indicating what the expected format here is would be useful.

Reply:
I added a comment.